includes/Modules/Analytics.php

```diff
@@ -595,33 +563,44 @@ protected function get_datapoint_services() {
+			'GET:accounts-properties-profiles' => array( 'service' => 'analytics' ),
+			'GET:anonymize-ip'                 => array( 'service' => '' ),
+			'GET:connection'                   => array( 'service' => '' ),
+			'POST:connection'                  => array( 'service' => '' ),
+			'POST:create-account-ticket'       => array(
+				'service'                => 'analyticsprovisioning',
 				'scopes'                 => array( 'https://www.googleapis.com/auth/analytics.provision' ),
-				'request_scopes_message' => __( 'You’ll need to grant Site Kit permission to create a new Analytics account on your behalf.', 'google-site-kit' ),
-			)
-		);
-
-		$map['POST:create-property'] = array_merge(
-			$map['POST:create-property'],
-			array(
+				'request_scopes_message' => __( 'You\'ll need to grant Site Kit permission to create a new Analytics account on your behalf.', 'google-site-kit' ),
+			),
+			'POST:create-profile'              => array(
+				'service'                => 'analytics',
 				'scopes'                 => array( 'https://www.googleapis.com/auth/analytics.edit' ),
-				'request_scopes_message' => __( 'You’ll need to grant Site Kit permission to create a new Analytics property on your behalf.', 'google-site-kit' ),
-			)
-		);
-
-		$map['POST:create-profile'] = array_merge(
-			$map['POST:create-profile'],
-			array(
+				'request_scopes_message' => __( 'You\'ll need to grant Site Kit permission to create a new Analytics view on your behalf.', 'google-site-kit' ),
+			),
+			'POST:create-property'             => array(
+				'service'                => 'analytics',
 				'scopes'                 => array( 'https://www.googleapis.com/auth/analytics.edit' ),
-				'request_scopes_message' => __( 'You’ll need to grant Site Kit permission to create a new Analytics view on your behalf.', 'google-site-kit' ),
-			)
+				'request_scopes_message' => __( 'You\'ll need to grant Site Kit permission to create a new Analytics property on your behalf.', 'google-site-kit' ),
+			),
+			'GET:internal-web-property-id'     => array( 'service' => '' ),
+			'POST:internal-web-property-id'    => array( 'service' => '' ),
+			'GET:goals'                        => array( 'service' => 'analytics' ),
+			'GET:profile-id'                   => array( 'service' => '' ),
+			'POST:profile-id'                  => array( 'service' => '' ),
+			'GET:profiles'                     => array( 'service' => 'analytics' ),
+			'GET:properties-profiles'          => array( 'service' => 'analytics' ),
+			'GET:property-id'                  => array( 'service' => '' ),
+			'POST:property-id'                 => array( 'service' => '' ),
+			'GET:report'                       => array( 'service' => 'analyticsreporting' ),
+			'GET:tag-permission'               => array( 'service' => '' ),
+			'GET:tracking-disabled'            => array( 'service' => '' ),
+			'POST:tracking-disabled'           => array( 'service' => '' ),
```

Comment:
As far as I can see `POST:tracking-disabled` does not actually exist. It was in the wrong place in the original array in regards to the method comments.

Reply:
Ok, removed.